kubernetes/terraform/public-keys/aks/id_rsa.pub

```diff
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDX81edxT9sUqi+vcMgykS0D5I8H02Tf8Oezg0hUAv1JFiP2l3wFMAI10jxcGzMi47+3VRxzrBOky56hKM68zmak+OleVRweMIp35VqgsISC8+bFlzTbWlVQQtMo7qxkoiKUUhwdYgb3W5Vtyf9QNyFn0M5ZSd51bf+MV0yotctPdsvE99k0hoXVsUDqeZiv+rnDQUgC1yEL1c6p+e4vjr4AvDV3sm4ZX048c09hokiEZ3/qeUW1gtDT8bJGsPtOzOFf9RekzGFR5fb/LCIDhqUsEvxV4xshKHTSVhCrzLFlbXdYyUOnstpQbY65BcThdlfCJmOTBU4woiO1XRrWZpPojUFXchj1k02CKEK5BQ2px5iERo4Ug8ef+n7g7ifo/SduoqXCJy6Zc1hL+nwn+U4WG9o/yn1e7vV+Iu7rrjVungD1L17dCYk8qWCSDlBe90QO3fQy3BaP/2WJ429ARtpkC4gYUanG2w0QXul0oYHzzujr3JMo4Jcp1PIUR1BAc8pHy/xCyta8DN6oVfyegQBZ68rO+iGv/cPb8uhdkVyycf8RvR03tZ0MDBvxDSCeD+wu1sLVRp/cquNQGi2hHTFuBPkgZBQp+cnLYMEZvhwiKvQ23pgsc2JUKOgoESPeyi2njL7mcarUdvXev7bIw+E0PYwWs70HzjPvIFRZC0VvQ== lakshanb@wso2.com
```

Comment:
Is it intentional to have this public keys maintained in the repo? Why exactly we need to keep them maintained here.

Reply:
Yes it's intentional

We are injecting these public keys to the VMs during creation to allow SSH into them. The private keys associated with these public keys are stored in an Azure key vault and can be used by developers to SSH into the VMs.

However, I agree that it is better to pass these as parameters from the pipeline. I'll check on the feasibility of doing that.